zephyr/lib/cpu.c

```diff
@@ -148,7 +148,7 @@ int cpu_enable_core(int id)
 	 * initialization. By reinitializing the idle thread, we would overwrite the kernel structs
 	 * and the idle thread stack.
 	 */
-	if (!IS_ENABLED(CONFIG_ADSP_IMR_CONTEXT_SAVE) ||
+	if (!defined(CONFIG_ACE) ||
```

Comment:
@mengdonglin looks good. Can we also update the comment above to call out the difference between cavs and ACE?

Reply:
@ranj063 Thanks for your review. I updated the comments in the code.